crates/bevy_audio/Cargo.toml

```diff
@@ -26,7 +28,9 @@ rodio = { version = "0.17", default-features = false }
 oboe = { version = "0.5", optional = true }
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-rodio = { version = "0.17", default-features = false, features = ["wasm-bindgen"] }
+rodio = { version = "0.17", default-features = false, features = [
```

Comment:
These "auto formatted line length enforcement changes" should be in a separate PR and should be coupled with CI validation. Otherwise we risk thrashing as two people use different local formatters with different settings. Lets keep this PR scoped to the change at hand.

Reply:
I'm doing that in this pr https://github.com/bevyengine/bevy/pull/10594.

Reply:
ok fixed